etc/helm/pachyderm/templates/kube-event-tail/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.kubeEventTail.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kube-event-tail
+    suite: pachyderm
+  name: kube-event-tail
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kube-event-tail
+      suite: pachyderm
+  template:
+    metadata:
+      labels:
+        app: kube-event-tail
+        suite: pachyderm
+    spec:
+      containers:
+        - name: kube-event-tail
+          env:
+            - name: DEBUG_ADDRESS
+              value: 0.0.0.0:8081
+            - name: NAMESPACE
+{{- if eq .Values.kubeEventTail.clusterScope true }}
+              value: ""
+{{ else }}
+              value: {{ .Release.Namespace }}
+{{- end }}
+          image: docker.io/jrockway/kube-event-tail:v0.0.6
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /healthz
+              port: debug
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          ports:
+            - containerPort: 8081
+              name: debug
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /healthz
+              port: debug
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          resources:
+            limits:
+              memory: 45Mi
```

Comment:
you can set a cpu limit of 1 here.  (non-integer cpu limits are problematic, random k8s.af article: https://erickhun.com/posts/kubernetes-faster-services-no-cpu-limits/; this is why pachd and opinionated-server include the `maxprocs` library, so we never run into that latency problem they describe)

Reply:
Done + I moved the resources into the chart in case they need to be configurable